### PR TITLE
Ultimate software origin policy

### PR DESCRIPTION
**Pull request by DavidePrincipi**
- drop NsReleaseLock prop and apply new repository configuration
- define RepoGpgCheckStatus prop

NethServer/dev#5704

### CODE REVIEW

File: root/etc/e-smith/db/configuration/defaults/sysconfig/RepoGpgCheckStatus [hunk 1]
```diff
@@ -0,0 +1 @@
+enabled
```
Comment (gsanchietti) [inline root/etc/e-smith/db/configuration/defaults/sysconfig/RepoGpgCheckStatus]: Could we remove this prop and simplify the code?

File: README.rst [hunk 2]
```diff
@@ -285,79 +285,69 @@ The event to expand the templates of all rpm which use TLS is ``tls-policy-save`
...
-Actually, this limitation is present for:
+.. warning::
 
-- ``epel``
-- ``centos-sclo-rh``
-- ``centos-sclo-sclo``
+    When a subscription is enabled the default repositories are disabled. See
+    :file:`/etc/yum.repos.d/subscription.repo`
 
+Repository GPG signature check
```
Comment (gsanchietti) [inline README.rst]: Remove this section.

File: root/etc/e-smith/templates/etc/yum/vars/nsrelease/00default [hunk 3]
```diff
@@ -8,7 +8,7 @@
        <$nsh>;
    };
 
-   our $release = '';
+   our $release = $defaultRelease;
```
Comment (gsanchietti) [inline root/etc/e-smith/templates/etc/yum/vars/nsrelease/00default]: Remove `/etc/yum/vars/nsrelease` template and add the standard config file inside nethserver-release.

Comment (DavidePrincipi) [reply root/etc/e-smith/templates/etc/yum/vars/nsrelease/00default]: if we remove that template, the automatic upgrade of subscriptions must be implemented by an action

**Pull request merged**